minio/policy.py

```diff
@@ -528,36 +528,50 @@ def _get_permissions(s, resource, object_resource, matched_resource,
 
 
 # Returns policy of given bucket name, prefix in given statements.
-def get_policy(statements, bucket_name, prefix=''):
```

Comment:
There isn't necessarily a problem with this function. After developing the `list_policy` method in `policy.py`, most of the code inside of `get_policy` in `policy.py` is unnecessary. It would be easier to just use the `list_policy` method to return a dictionary of policies and do a `dict.get()` to find any policies that match the prefix supplied as a parameter in `get_policy`